### DISCUSSION TITLE
Trait type issue

### DISCUSSION THREAD

**Discussion by charleshys** [nftchef/art-engine#73] [Help + Q&A] [answered]
![image](https://user-images.githubusercontent.com/96027543/155938833-08a43c80-4cb1-42a8-977e-02c1ebdceb71.png)

I am having an issue with my metadata, I don't know why I have the following trait type and value on my metadata

{
      "trait_type": "C:\\Users\\charl\\Desktop\\NTC\\Development\\art-engine-nested-folder-structure\\art-engine-nested-folder-structure\\layers",
      "value": "Hat"
    },
    
  is there anyone know how to solve this?

**Comment by nftchef** [comment, marked as answer]
This is 99% of the time due to an image in the root folder without a weight, and that is usually the hidden `Desktop.ini` file on Windows. Do a search for that in your layers folder, and in layers/Hat folder and delete the file, or remove/change any file without a `#` weight

**Reply by charleshys** [reply]
Thanks a lot! problem solved!
